docs/docs/query-filters.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Query Filters with GraphQL in Gatsby
+---
+
+> This documentation is up to date as of `gatsby@2.23.0`.
+
+## Summary
+
+Gatsby stores all data loaded during the source-nodes phase in Redux and it allows you to write GraphQL queries to query that data. This data, stored as individual "nodes", can be searched through using a query language that is a subset of [MongoDb queries](https://docs.mongodb.com/manual/reference/operator/query/).
+
+When you write a GraphQL query, the `filter` parts will be processed and Gatsby will return all the nodes that match each of the given filters. GraphQL does the rest.
+
+### History and Sift
+
+For a long time Gatsby used the [sift.js](https://github.com/crcn/sift.js) library through which you can use [MongoDb queries](https://docs.mongodb.com/manual/reference/operator/query/) in JavaScript.
+
+Unfortunately Sift did not align with how Gatsby used it and so a custom system was written to slowly replace it. This system was called "fast filters" and as of gatsby@2.23.0 (June 2020) the Sift library is no longer used.
+
+## Query Language
+
+The syntax and API used by the filters is based on [the MongoDB query syntax](https://docs.mongodb.com/manual/reference/operator/query/) but keep in mind only a subset of comparators is supported in Gatsby.
+
+In general a filter has a "filter path" component ending with a "comparator", and a "filter value". It's a way of programmatically asking "show me all nodes where `node[i].a.b.c` leads to a value less than `5`".
+
+- path: an exact property path when traversing an object (`{ a: { b: { c: { op: x } } } }` has path `a`, `a.b`, and `a.b.c`)
+- value: the needle to compare against the value at the end of a path
+- comparator: a way to tell the system how to compare the given filter value to the resulting value at the end of the path. Should it be equal, lower than, or match in a special way?
+
+Filters are generally applied to a subset of all nodes, matching a specific `node.internal.type` value, like `` `PageData` `` or `` `markdownRemark` ``.
```

Comment:
How do you usually handle a quoted string in a code span?

We use backticks for strings as per our styleguide. To have code with backticks you start and end with more backticks. So that's why it's there.

Reply:
on other places in the doc you use normal ones 🤔 
```markdown
`'name'`
```